src/beautifiers/beautysh.coffee

```diff
@@ -0,0 +1,15 @@
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class BashBeautify extends Beautifier
+  name: "beautysh"
+  link: "https://github.com/bemeurer/beautysh"
+  options: {
+    Bash:
+      indent_size: true
```

Comment:
I'm gonna try to debug `docs/index.coffee` but it looks like it does not fail from the output.